deploy/materialized_view_contact.sql

```diff
@@ -0,0 +1,37 @@
+-- Deploy ggircs:materialized_view_contact to pg
+-- requires: table_ghgr_import
+
+begin;
+
+create materialized view ggircs_swrs.contact as (
+  with x as (
+    select _ghgr_import.xml_file as source_xml,
+           _ghgr_import.id         as ghgr_import_id
+    from ggircs_swrs.ghgr_import as _ghgr_import
+    order by ghgr_import_id asc
+  )
+  select ghgr_import_id, contact_details.*
+  from x,
+       xmltable(
+           '//Contact[not(ancestor::ConfidentialityRequest)]'
+           passing source_xml
+           columns
+                facility_id numeric(1000,0) path './ancestor-or-self::Contact/ancestor::ReportData/ReportDetails/FacilityId[normalize-space(.)]',
+                path_context varchar(1000) path 'name(./ancestor::VerifyTombstone|./ancestor::RegistrationData)',
```

Comment:
Added a path context to show whether the contact was pulled under RegistrationData or VerifyTombstone, good approach or should we just pull from one source?